### PR TITLE
[MHV-56085, MHV-56097, MHV-56162] V3 Component Fixes SM mhv

### PR DESCRIPTION
**Pull request by vsaleem**
>>
> ### Summary
>
- This update converts the 'Edit Folder Name' modal to uswds true and corrects margin spacing between the buttons on the modal
- Also, margin spacing corrections were made to the CrisisLineConnectionButton located inside the expandable-alert on the ComposeForm page and,
- Folder Name text was corrected to keep text aligned with the icon, located on the Custom Folder List screen.
>
> ### Related issue(s)
>
> #### [[MHV-56162]](https://jira.devops.va.gov/browse/MHV-56162) Mobile view buttons are too close together (Edit Folder Name Modal) (**_This ticket is no longer needed._**)
> #### [[MHV-56097]](https://jira.devops.va.gov/browse/MHV-56097) Add margin after and before VCL button (Compose Form Expandable Alert)
> #### [[MHV-56085]](https://jira.devops.va.gov/browse/MHV-56085) Text should not start on next line if folder name has no space (Custom Folder List View)
>
> ### Testing done
>
- _Manual testing_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_
>
> ### Screenshots
>
_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._
>
|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |
image
image
|
| Desktop |        |       |
>
> ### What areas of the site does it impact?
>
*(Describe what parts of the site are impacted **if** code touched other areas)*
>
> ### Acceptance criteria
>
> ### Quality Assurance & Testing
>
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed
>
> ### Error Handling
>
- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)
>
> ### Authentication
>
- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user
>
- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions
>
> ## Requested Feedback
>>
(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/mhv/secure-messaging/components/FoldersList.jsx [hunk 1]
```diff
@@ -18,16 +18,18 @@ const FoldersList = props => {
...
-                  {folder.id === Folders.DELETED.id
-                    ? Folders.DELETED.header
-                    : folder.name}{' '}
-                  {showUnread &&
-                    folder.unreadCount > 0 &&
-                    folder.id !== Folders.DRAFTS.id &&
-                    `(${folder.unreadCount} unread messages)`}
-                </span>
+                <div className="icon-span-container">
+                  <i className="fas fa-folder fa-lg" aria-hidden="true" />
```
[the remark this answers is not in the document]
Comment (rmessina1010) [reply src/applications/mhv/secure-messaging/components/FoldersList.jsx:22]: would <va-icon> not be appropriate here?
https://design.va.gov/about/developers/using-web-components#how-to-migrate-from-font-awesome-to-va-icon

**Review by alex-morgun-1 — CHANGES_REQUESTED** [review]
since the new `src/applications/mhv-secure-messaging` directory is up, let's migrate this code over to the new directory

File: src/applications/mhv/secure-messaging/components/FoldersList.jsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ const FoldersList = props => {
                     folder.unreadCount > 0 &&
                     folder.id !== Folders.DRAFTS.id &&
                     `(${folder.unreadCount} unread messages)`}
-                </span>
+                </span>{' '}
```
Comment (oleksii-morgun) [inline src/applications/mhv/secure-messaging/components/FoldersList.jsx:30]: just checking if this is whitespace is needed

**Review by alex-morgun-1 — DISMISSED** [review]
Approving pending couple items to circle around

File: src/applications/mhv/secure-messaging/components/ManageFolderButtons.jsx [hunk 3]
```diff
@@ -224,7 +224,7 @@ const ManageFolderButtons = props => {
           data-dd-action-name="Save Rename Folder Button"
         />
         <va-button
-          uswds={false}
+          uswds="false"
```
[the remark this answers is not in the document]
Comment (vsaleem) [reply src/applications/mhv/secure-messaging/components/ManageFolderButtons.jsx:227]: We don't need to keep this as false. This change will be covered in @KyleCardwell's [PR](https://github.com/department-of-veterans-affairs/vets-website/pull/28915).

File: src/applications/mhv/secure-messaging/components/FoldersList.jsx (hunk 2 above)
Comment (vsaleem) [reply src/applications/mhv/secure-messaging/components/FoldersList.jsx:30]: You're right. This is not needed.

**Review by alex-morgun-1 — APPROVED** [review]
Great 👍🏻

**Pull request merged**